### PR TITLE
Stop ignoring **/_.pdb files in Docker containers

### PR DESCRIPTION
**Pull request by alnikola**
Networking stress tests running on Linux are failing in master at building runtime inside container due to a missing `_.pdb` files (e.g. `/repo/src/coreclr/src/.nuget/_.pdb`) This PR makes Docker to include those files into containers.

### CODE REVIEW

**Review by aik-jahoda — APPROVED** [review]

File: .dockerignore [hunk 1]
```diff
@@ -61,6 +61,7 @@
 **/*.obj
 **/*.pch
 **/*.pdb
+!**/_.pdb
```
Comment (aik-jahoda) [inline .dockerignore:64]: It worth a comment with explanation why one kind of pdb is ignored while other allowed.

**Comment by safern** [comment]
I think I added this change to: https://github.com/dotnet/runtime/pull/35011 -- but fine to split it in a separate PR.

**Review by safern — APPROVED** [review]

**Comment by alnikola** [comment]
Closing it as obsolete since #35011 was merged.

**Pull request closed without merging**